app/views/display_ads/for_display.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if @display_ad %>
+  <% if user_signed_in? %>
+    <%= render partial: "shared/display_ad", locals: { display_ad: @display_ad, data_context_type: DisplayAdEvent::CONTEXT_TYPE_ARTICLE } %>
+  <% else %>
+    <% cache("display-ad#{@display_ad.placement_area}-#{@article.id}", expires_in: 15.minutes) do %>
```

Comment:
Some Fragment Caching? Any thoughts here?

Reply:
```suggestion
    <% cache([@article, @display_ad.placement_area], expires_in: 15.minutes) do %>
```
The file name, `views/display_ads/for_display` will be automatically part of the cache name if you do the following. Another added benefit is if this file is updated, the cache will be purged. Or maybe this isn't what you want, try it out!

ref https://api.rubyonrails.org/classes/ActionView/Helpers/CacheHelper.html#method-i-cache

Reply:
Nice! Thank you @maestromac! I'll take it for a spin!

Reply:
`Write fragment views/display_ads/for_display:96901e6373ce5016e8a4ab0e9cace74c/articles/60-20230330074328502102/post_comments`

This is great!